### PR TITLE
Update UG, DG for sort

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Ardentsoul — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
...
-* Use sna/ to sort name in ascending order.
-* Use snd/ to sort name in descending order.
-* Use ssa/ to sort status in ascending order.
-* Use ssd/ to sort status in descending order.
-* If there are no prefix, it will sort by name in ascending order.
-* If both prefixes for ascending and descending are used for the same attribute, it will only sort by ascending.
-* If both prefixes for name and status are used, by default, it will sort by name then status.
-* Use so/ to sort by status then name.
+:exclamation:**Caution:**<br>
+*You cannot undo the sort once you applied it!
```
Comment (arcornior) [inline docs/UserGuide.md:368]: Minor grammatical error here
Should either be `once you have applied it!` or `once you apply it!`

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -318,14 +318,13 @@ The following activity diagram summarizes what happens when a user executes a va
 ### Sort command feature
 
 #### What it does
-Sorts the list of shows according to the user's input prefix. If the input contains no prefixes, sort will default to organizing the shows in ascending order by name. If both prefixes for ascending and descending are used, it will only sort by ascending. If both prefixes for name and status are used, by default it will sort by name then by status.
-
+Sorts the list of shows according to the user's input prefix. If the input contains no prefixes, sort will default to organizing the shows in ascending order by name. The format for sort is `sort [n/ORDER] [s/ORDER] [t/ORDER] [r/ORDER] [so/SEQUENCE]`. `ORDER` must be either "asc" or "dsc". If two or more of the above prefixes are being used, sort will prioritise sorting by name then status then rating then tag. Use `so/` to change the priority. `SEQUENCE` must contain the full name of the prefix used and the user have to list it in the sequence in the order which sort prioritise.
```
Comment (arcornior) [inline docs/DeveloperGuide.md:321]: I would also rephrase
`sorting by name then status then rating then tags`  to
`sorting by name, then status, followed by rating, and finally tags`
just to avoid the repeating of then.
At the very least I think the usage of commas to separate the different sorting conditions should be included. Just a nitpick though.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
...
+* Use `s/` if you want to sort by status
+* Use `t/` if you want to sort by tag
+* Use `r/` if you want to sort by rating
+* For the above prefixes, you have to specify the `ORDER` right after the prefix.
+* `ORDER` is the order you want to sort the list by! Type `asc` to sort by ascending or type `dsc` to sort by descending! 
+* If two or more of the above prefixes are being used, sort will prioritise sorting by name then status then rating then tag.   
+* Use `so/` if you want to reorder the priority and state the `SEQUENCE`
+* List the `SEQUENCE` by listing out the full name of the prefix used in the order you want.
+* For example: you want to sort by rating then tag then status then name, `SEQUENCE` will be "RatingTagStatusName"
+* If you are confused, don't worry! Just do not use any prefix, we will help you sort it by name in ascending order!
```
Comment (arcornior) [inline docs/UserGuide.md:383]: Nice touch! I like how you provide the user with an easy to use option!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
...
+* Use `so/` if you want to reorder the priority and state the `SEQUENCE`
+* List the `SEQUENCE` by listing out the full name of the prefix used in the order you want.
+* For example: you want to sort by rating then tag then status then name, `SEQUENCE` will be "RatingTagStatusName"
+* If you are confused, don't worry! Just do not use any prefix, we will help you sort it by name in ascending order!  
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about Sort:**<br>
+* By not entering any prefixes, it will sort by name in ascending order.
+* Enter the right amount prefixes! For example: if you only sort by status, use only `s/`.
```
Comment (arcornior) [inline docs/UserGuide.md:389]: Should this be `Enter the right amount of prefixes!`?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
...
+* If you are confused, don't worry! Just do not use any prefix, we will help you sort it by name in ascending order!  
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about Sort:**<br>
+* By not entering any prefixes, it will sort by name in ascending order.
+* Enter the right amount prefixes! For example: if you only sort by status, use only `s/`. 
+* The `ORDER` must be asc or dsc!
+* The `SEQUENCE` must contain the full name of all the prefixes used!
+* For `SEQUENCE`, if more than one of the same full name is being used, it will only consider the first instance of the full name. For example: "RatingTagRating" will sort by rating then tag.
```
Comment (arcornior) [inline docs/UserGuide.md:392]: Maybe we can change full name to prefix? This will follow the way you address them in the next line.
I'd like to slightly rephrase the first sentence to
```
For `SEQUENCE`, if the same prefix is being used multiple times, it will only consider the first instance of the prefix.
```
Might just be me, but I feel that this will make it easier for the reader to understand the meaning of the sentence.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
...
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about Sort:**<br>
+* By not entering any prefixes, it will sort by name in ascending order.
+* Enter the right amount prefixes! For example: if you only sort by status, use only `s/`. 
+* The `ORDER` must be asc or dsc!
+* The `SEQUENCE` must contain the full name of all the prefixes used!
+* For `SEQUENCE`, if more than one of the same full name is being used, it will only consider the first instance of the full name. For example: "RatingTagRating" will sort by rating then tag. 
+* If multiple same prefixes are being used, only the last prefix will be considered by the program! For example: `sort n/asc n/dsc` will sort name by descending order.
```
Comment (arcornior) [inline docs/UserGuide.md:393]: Likewise, I would rephrase
`If multiple same prefixes are being used,` to `If the same prefix is being used multiple times,`

File: docs/UserGuide.md [hunk 7]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
...
+</div>
+
+**Example & Output:** `sort`
+
+<img src="images/Sort_UI.png">
+
+**Example & Output:** `sort t/dsc r/asc`
+
+<img src="images/Sort_UI2.png">
```
Comment (arcornior) [inline docs/UserGuide.md:402]: Not sure why but the command result box shows that a show has been deleted!
Could you check if this is the right output?

**Review by arcornior — CHANGES_REQUESTED** [review]
Great job on the documentation! Just a few grammatical errors here and there.

File: docs/UserGuide.md (hunk 1 above)
Comment (lowkaiwei98) [reply docs/UserGuide.md:368]: ok

File: docs/UserGuide.md [hunk 8]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
 
 ---
 
-### Sorting the shows
-[TO CHANGE AFTER REWORK]
+### Sorting the shows `sort`
 
-Sort shows based on the input prefix.
+Your show list is in mess! Do you need a way to organise the list?
```
[the remark this answers is not in the document]
Comment (lowkaiwei98) [reply docs/UserGuide.md:364]: ok

File: docs/UserGuide.md (hunk 5 above)
Comment (lowkaiwei98) [reply docs/UserGuide.md:392]: If you use prefix it can refer to "so/" too, might cause further confusion?

File: docs/UserGuide.md (hunk 7 above)
Comment (lowkaiwei98) [reply docs/UserGuide.md:402]: fixed

File: docs/UserGuide.md (hunk 5 above)
Comment (arcornior) [reply docs/UserGuide.md:392]: Ahh that's true. Maybe it'd be more correct to use the term sort conditions?
`if a sort condition is being used multiple times` instead? Would this help?

Comment (lowkaiwei98) [reply docs/UserGuide.md:392]: ok

File: docs/UserGuide.md [hunk 9]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
...
+* For example: you want to sort by name, then status, followed by rating, and finally tags, `SEQUENCE` will be "RatingTagStatusName"
+* If you are confused, don't worry! Just do not use any prefix, we will help you sort it by name in ascending order!  
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about Sort:**<br>
+* By not entering any prefixes, it will sort by name in ascending order.
+* Enter the right amount of prefixes! For example: if you only sort by status, use only `s/`. 
+* The `ORDER` must be asc or dsc!
+* The `SEQUENCE` must contain the full name of all the sort condition used!
```
Comment (arcornior) [inline docs/UserGuide.md:392]: I think `sort condition` should be `sort conditions` here.

File: docs/UserGuide.md [hunk 10]
```diff
@@ -359,20 +359,51 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
...
+Format: `sort [n/ORDER] [s/ORDER] [t/ORDER] [r/ORDER] [so/SEQUENCE]…​`
+
+`Sort` will sort according to the [prefixes](#command-structure)**
+* Use `n/` if you want to sort by name 
+* Use `s/` if you want to sort by status
+* Use `t/` if you want to sort by tag
+* Use `r/` if you want to sort by rating
+* For the above prefixes, you have to specify the `ORDER` right after the prefix.
+* `ORDER` is the order you want to sort the list by! Type `asc` to sort by ascending or type `dsc` to sort by descending! 
+* If two or more of the above prefixes are being used, sort will prioritise sorting by name then status then rating then tag.
```
Comment (arcornior) [inline docs/UserGuide.md:380]: Sorry, I think I missed this the last time I was reviewing
`sort will prioritise sorting by name, then status, followed by rating, and finally tags.`

**Review by arcornior — CHANGES_REQUESTED** [review]
LGTM! Just a couple of nitpicks.

**Review by arcornior — APPROVED** [review]
LGTM!
